pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1263,6 +1257,14 @@ func (b *MacvtapBindMechanism) discoverPodNetworkInterface() error {
 	return nil
 }
 
+func (b *MacvtapBindMechanism) getIfaceMACAddr() string {
```

Comment:
nit: I think you can drop the `get` prefix and be more specific to what mac address we fetch here (e.g. `podIfaceMAC()`).
Personally I get lost here between the all the interfaces (bridge, tap, veth, vnic).

Reply:
+1, https://golang.org/doc/effective_go#Getters

Reply:
> nit: I think you can drop the `get` prefix and be more specific to what mac address we fetch here (e.g. `podIfaceMAC()`).
> Personally I get lost here between the all the interfaces (bridge, tap, veth, vnic).

I had it named like that because there's no veth nor bridge involved in macvtap; only a macvlan side (pod) plus a character device which will be used to create an emulated vnic on the VM side. 

I never imagined it would be confusing to know to which interface this code was referring to.

But hey, let's make the reader's life easier. Renamed it according to your suggestion; hope that the code became more explicit.